src/DynamoPackages/Package.cs

```diff
@@ -29,7 +29,7 @@ public string Name
         }
     }
 
-    public class Package : NotificationObject, ILogSource
+    public class Package : NotificationObject, ILogSource, IPackage
```

Comment:
API break?

Reply:
@aparajit-pratap @mjkkirschner What do you think about adding a `PackageInfo` class to avoid having to pass `Package`/`IPackage` objects around and to avoid the API break? See this commit: https://github.com/DynamoDS/Dynamo/pull/9723/commits/21181818535dc800a378c8e4af86116429348a24